src/main/java/ca/corefacility/bioinformatics/irida/ria/web/ajax/projects/ProjectUserGroupsAjaxController.java

```diff
@@ -0,0 +1,105 @@
+package ca.corefacility.bioinformatics.irida.ria.web.ajax.projects;
+
+import java.util.List;
+import java.util.Locale;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import ca.corefacility.bioinformatics.irida.exceptions.ProjectWithoutOwnerException;
+import ca.corefacility.bioinformatics.irida.model.user.group.UserGroup;
+import ca.corefacility.bioinformatics.irida.ria.web.ajax.dto.NewMemberRequest;
+import ca.corefacility.bioinformatics.irida.ria.web.ajax.dto.ProjectUserGroupsTableModel;
+import ca.corefacility.bioinformatics.irida.ria.web.models.tables.TableRequest;
+import ca.corefacility.bioinformatics.irida.ria.web.models.tables.TableResponse;
+import ca.corefacility.bioinformatics.irida.ria.web.services.UIProjectUserGroupsService;
+
+/**
+ * UI Ajax Controller for handling project user groups
+ */
+@RestController
+@RequestMapping("/ajax/projects/{projectId}/user-groups")
+public class ProjectUserGroupsAjaxController {
+	private final UIProjectUserGroupsService service;
+
+	@Autowired
+	public ProjectUserGroupsAjaxController(UIProjectUserGroupsService service) {
+		this.service = service;
+	}
+
+	/**
+	 * Get a table page of {@link ProjectUserGroupsTableModel}
+	 *
+	 * @param projectId Identifier for a project
+	 * @param request   {@link TableRequest} details about the current page of the table
+	 * @return {@link TableResponse}
+	 */
+	@RequestMapping("")
+	public ResponseEntity<TableResponse<ProjectUserGroupsTableModel>> getProjectUserGroups(@PathVariable Long projectId,
+			@RequestBody TableRequest request) {
+		return ResponseEntity.ok(service.getUserGroupsForProject(projectId, request));
+	}
+
+	/**
+	 * Remove a user group from a project
+	 *
+	 * @param projectId Identifier for a project
+	 * @param groupId   Identifier for an user group
+	 * @param locale    current users locale
+	 * @return message to user about the result of removing the user group
+	 */
+	@RequestMapping(value = "", method = RequestMethod.DELETE)
+	public ResponseEntity<String> removeUserGroupFromProject(@PathVariable long projectId, @RequestParam long groupId,
+			Locale locale) {
+		return ResponseEntity.ok(service.removeUserGroupFromProject(projectId, groupId, locale));
```

Comment:
Just a general question. Even if there is an error deleting the usergroup would we still want it to return a `ResponseEntity.ok`? Or should we set the status as a bad request with the error message?

Reply:
No but it should throw it's own error at that point and not actually return here.

Reply:
Sounds good!